src/App.tsx

```diff
@@ -10,33 +11,24 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
-
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+  const [selectedTabTitle, setSelectedTabTitle] = useState('Tab 1');
+  const [selectedTabContent, setSelectedTabContent] = useState('Some text 1');
 
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
+  const selectedTab = (id: string, title: string, content: string) => {
+    setSelectedTabId(id);
+    setSelectedTabTitle(title);
+    setSelectedTabContent(content);
+  };
```

Comment:
👍 But now you will accept a tab from your child component, so you can use your setSelectedTabId accordingly. (It is also required for one of the tests to get a tab from the child instead of tab.id for example)